dts/arm/nxp/nxp_lpc55S3x_common.dtsi

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 NXP
+ * Copyright 2022,2024 NXP
```

Comment:
need space after comma

Reply:
thanks for your reminder, I will update it.